torch_frame/data/multi_nested_tensor.py

```diff
@@ -224,6 +231,29 @@ def stack(xs: List['MultiNestedTensor'],
         else:
             raise NotImplementedError
 
+    @staticmethod
+    def cat(xs: List['MultiNestedTensor'],
+            dim: int = 0) -> 'MultiNestedTensor':
+        if dim < 0:
+            dim += xs[0].ndim
+        if dim == 0:
+            num_rows = sum(x.num_rows for x in xs)
+            num_cols = xs[0].num_cols
+        elif dim == 1:
+            num_rows = xs[0].num_rows
+            num_cols = sum(x.num_cols for x in xs)
+        else:
+            raise NotImplementedError
+        values = torch.cat([x.values for x in xs], dim=0)
+        accum_offset = 0
+        offset = []
+        for x in xs:
+            offset.append(x.offset + accum_offset)
+            accum_offset = accum_offset + x.offset[-1]
```

Comment:
This logic is only applicable to `dim=0`.

Reply:
Right, moved to `dim=0`